### PR TITLE
fix: Support TLS for kafka connection

### PR DESCRIPTION
**Pull request by yhmo**
#27977, https://github.com/milvus-io/milvus/issues/30709

Add extra configurations in milvus.yaml to pass certificates for kafka.

### CODE REVIEW

**Comment by yhmo** [comment]
rerun ut

File: configs/milvus.yaml [hunk 1]
```diff
@@ -126,6 +126,12 @@ pulsar:
 #   saslMechanisms: PLAIN
 #   securityProtocol: SASL_SSL
 #   readTimeout: 10 # read message timeout in seconds
+#   ssl:
```
Comment (jaime0815) [inline configs/milvus.yaml:129]: Using TLS as an option name would be better, SSL options already exist under kafka?

**Comment by yhmo** [comment]
rerun ut

**Pull request merged**